draft-irtf-cfrg-opaque.md

```diff
@@ -481,14 +480,23 @@ see {{ake-protocol}}. 3DH assumes a prime-order group as described in
 
 # Protocol Overview {#protocol-overview}
 
-OPAQUE consists of two stages: registration and authenticated key exchange.
-In the first stage, a client registers its password with the server and stores
-its credential file on the server. In the second stage the client recovers its
-authentication material and uses it to perform a mutually authenticated key
-exchange. For both stages, client and server agree on a configuration, which
+OPAQUE consists of two stages: registration and online authentication.
+In the first stage, a client uses its password to generate authentication
+material that server stores in a credential file.
+In the second stage the client recovers its authentication material from server
+and uses it to perform a mutually authenticated key exchange.
+
+## Setup
+Previously to both stages, client and server agree on a configuration, which
```

Comment:
```suggestion
and uses it to perform a mutually authenticated key exchange. For both stages, client and 
server agree on a configuration, which
```

Reply:
I feel a lack of a section called Setup, like any other protocols has.

Reply:
I don't think it's necessary.

Reply:
I think it is necessary. Shall we move this discussion to the list?

Reply:
No, I don't think so. This is an editorial consideration.

Reply:
Reverted and will be discussed in #289